src/config/messages.js

```diff
@@ -108,6 +108,10 @@ const PROMPT_DELETE_SPACE_MESSAGE =
   'Are you sure you want to delete this space?';
 const PROMPT_CLEAR_USER_INPUT_MESSAGE =
   'Are you sure you want to clear all of the user input in this space?';
+const ERROR_INSTALL_APP_UPGRADE_MESSAGE =
+  'An unexpected error occured while downloading or installing the upgrade.';
+const UPDATE_AVAILABLE_MESSAGE =
+  'A new version of Graasp Desktop is available. Do you want to quit the application and install it now?';
```

Comment:
Side question: did we figure out how to include translations here?

Reply:
In the PR done in graasp-insights involving toastr, the translation is mainly done in this file, but essentially we use the `i18n.t` function in the toastr on the message.